### PR TITLE
added in fourth item in feature box and adjusted mobile responsiveness

### PR DESCRIPTION
**Pull request by chanonroy**
screen shot 2017-01-03 at 5 48 44 pm

Looking to contribute more, holla if you need more help

### CODE REVIEW

**Comment by contra** [comment]
Copy is duplicated between this and the first item. You can revert the copy for the first item to what it used to be to rectify that.

**Comment by chanonroy** [comment]
Ah. Just noticed now. 

**Re: first item's old copy**
Is this the copy you're referring to? (from the [marvelapp](https://marvelapp.com/gj54hh/screen/14587285) mockup)
`"By preferring code over configuration, gulp keeps things simple and makes complex tasks manageable"`

Looking at the overall picture now, I can see why the 'Easy to Learn' feature item may have been omitted. Let me know if this is still a worthwhile effort. Otherwise, I can pick up on an official 'help wanted' issue.

**Comment by contra** [comment]
@chanonroy Yep, that copy. Title too. I think they're different enough to have both.

File: css/main.css [hunk 1]
```diff
@@ -66,6 +66,7 @@ body {
 }
 .subheading + .paragraph {
     margin-top: 10px;
+    padding: 0 30px 0 30px;
```
Comment (chanonroy) [inline css/main.css]: (optional) - makes padding for text copy look similar to the mockup

**Comment by contra** [comment]
@chanonroy  It seems like the latest changes updated the existing copy to the old stuff, but didn't add the new section to replace the lost copy?

**Comment by chanonroy** [comment]
@contra - it didn't carry over from the previous PR commit made earlier? On my fork the new section with lost copy is still there. Did it have to do with the way that the latest commit updated the PR?

**Comment by contra** [comment]
Hmm, looks a little funky:

image

**Comment by chanonroy** [comment]
Hmm. It should not look like that.

**Comment by chanonroy** [comment]
@contra - this is using the updated main.css, yes?

**Comment by contra** [comment]
@chanonroy I merged your whole PR so all the changes in here were deployed.

**Comment by chanonroy** [comment]
commit

I'm looking at the merge. I could be wrong, but I feel like every 'remove' line in red (#1) is something that I added, and that every 'add' line in green (#2) is something that I removed.

Something funky going on here.

**Comment by chanonroy** [comment]
Worse comes to worse, we can always revert it to the original with 3 feature boxes. Lots of effort for very minimal changes.

**Comment by contra** [comment]
@chanonroy Reverted it while you work it out.

**Pull request merged**